lib/udf_findhotel_utils.rb

```diff
@@ -10,22 +10,25 @@ class UdfFindhotelUtils
             if not room:
               return None
             else:
-              adults = 0
-              children = 0
-              rooms = room.split('|')
-              for r in rooms:
-                ch = r.split(':')
-                adults += int(ch[0])
-                if len(ch) > 1:
-                  children += len(ch[-1].split(','))
+              try:
```

Comment:
catching _all_ exceptions like this and simply silencing them is not a good practice. Instead you should catch exactly the exceptions you expect, and log everything:

```
import logging

logger = logging.getLogger()

... your code

    try:
        adults  += int(ch[0])
     except (ValueError, IndexError):
         logger.error("Invalid room specification: %s" % room)
         return


      ....

      try:
          children += len(ch[-1].split(','))
      except IndexError:
          logger.error("Invalid room specification: %s" % room)
          return
```